code/src/index.css

```diff
@@ -1,13 +1,189 @@
-body {
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@400;700&display=swap');
+
+* {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0 auto;
+  font-family: 'Roboto Mono', monospace;
+  background: #fcf9f9;
+}
+
+.main-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.thought-form, .thought-card {
+  width: 600px;
+  border: 2px solid rgba(0, 0, 0, 0.479);
+  margin-bottom: 40px;
+  box-shadow: 5px 5px;
+  padding: 15px;
+  background: #f8f8f8;
+}
+
+.thought-form {
+  height: 240px;
+  margin-top: 70px;
+}
+
+.form-title {
+  font-size: 20px;
+  font-weight: 700;
+  margin-bottom: 15px;
+}
+
+.form-input {
+  width: 100%;
+  height: 70px;
+  border: 1px solid rgba(0, 0, 0, 0.322);
+  overflow-wrap: break-word;
+  padding: 10px;
+  font-size: 16px;
+  resize: none;
+}
+
+.the-count {
+  text-align: right;
+}
+
+.black-counter {
+  color: #000;
+}
+
+.red-counter {
+  color: rgb(221, 0, 0);
+}
+
+.form-button {
+  width: 260px;
+  height: 40px;
+  border: none;
+  background-color: #ffadad;
+  border-radius: 40px;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  padding: 0 20px;
+  font-size: 16px;
+  margin: 15px 0;
+  cursor: pointer;
+  outline: none;
+  color: #000;
+}
+
+.form-button:hover {
+  background-color: #fdcfcf;
+}
+
+.thought-card {
+  height: 180px;
+  word-wrap: break-word;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.thought-message {
+  font-size: 20px;
+}
+
+.likes-time-wrapper {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 8px;
+
+}
+
+.likes-heart-wrapper {
+  display: flex;
+}
+
+.heart-button-no-likes {
+  padding: 15px;
+  font-size: 20px;
+}
+
+/* if likes > 0 */
+.heart-button-liked {
+  padding: 15px;
+  font-size: 20px;
+  background-color: #ffadad;
+  border-radius: 50%;
+}
+
+.heart-button {
+  border: none;
+  outline: none;
+  background-color: #f8f8f8;
+  cursor: pointer;
+}
```

Comment:
I really like👍 your likes button (redundant, isn't it?) But still, it looks neat with the right shape and colors and works great. I entertained myself clicking it for a while. 😆